src/components/toast/toast.tsx

```diff
@@ -56,6 +63,11 @@ export class BdsToast implements ComponentInterface {
   /**
    * Event used to execute some action when the action button on the toast is clicked
    */
+  @Prop() position: PositionType = 'bottom-left';
+  /**
+   * The toast position on the screen. Can be one of:
+   * 'top-right', 'top-left', 'bottom-right', 'bottom-left' (default value);
```

Comment:
this comment needs to be above the PositionType prop, it is inverted with the @Event toastButtonClick